src/main/java/com/salesforce/dynamodbv2/mt/context/impl/MtAmazonDynamoDbContextProviderImpl.java

```diff
@@ -28,9 +28,9 @@ public void setContext(String tenantId) {
     }
 
     @Override
-    public Optional<String> getContextOpt() {
-        String value = getContextMap().get(CONTEXT_KEY);
-        return value == null || value.trim().isEmpty() ? Optional.empty() : Optional.of(value);
+    public String getContext() {
```

Comment:
Before this change we only supported “empty” context for scans and records. Now we support it for all methods. Is that correct? If so, would be good to run this by Mike, since he had some feedback on the original PR that introduced the optional.